packages/database/src/migrations/20210503024117-AddLaEocMapOverlayDengueCasesByWeekDistrict-modifies-data.js

```diff
@@ -0,0 +1,105 @@
+('use strict');
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+// Note: Laos Eoc District is a sub_district entity
+const mapOverlay = {
+  id: 'LAOS_EOC_Dengue_Cases_By_Week_District',
+  name: 'Total Weekly Dengue Cases by District',
+  userGroup: 'Laos EOC User',
+  dataElementCode: 'NCLE_Disease_Name',
+  isDataRegional: false,
+  measureBuilderConfig: {
+    programCodes: ['NCLE_Communicable_Disease'],
+    dataSourceType: 'custom',
+    dataElementCode: 'NCLE_Disease_Name',
+    entityAggregation: {
+      dataSourceEntityType: 'facility',
+      aggregationEntityType: 'sub_district',
+      aggregationOrder: 'BEFORE',
```

Comment:
Technically there's no need for an aggregation order here since this is the only aggregation being performed, but I'm not sooo nitpicky that I'd hold you up on that 😝

Reply:
Good to know, I'll remove it.